src/ompl/geometric/planners/est/src/BiEST.cpp

```diff
@@ -57,15 +57,12 @@ void ompl::geometric::BiEST::setup()
 {
     Planner::setup();
 
-    if (maxDistance_ < 1e-3)
```

Comment:
Why remove the if statement? Now the selfconfig code is run each time setup() is called and overrides any user-specified range setting.

Reply:
Hm, maybe this if statement should actually be added to, e.g., ompl::geometric::EST::setup (and maybe other planners).

Reply:
From my understanding selfConfig for the Range only overrides if the parameter is set at less than 0 see at [line91](http://ompl.kavrakilab.org/SelfConfig_8cpp_source.html).   This is how it is done in all the planners that use the range parameter e.g, RRT, RRTConnect  so I removed it for consistency.  It seems redundant to have this check in both places. I don't know were they should be though